README.md

```diff
@@ -127,14 +129,18 @@ require('smart-splits').setup({
   -- this functionality is only supported on tmux and Wezterm due to kitty
   -- not having a way to check if a pane is zoomed
   disable_multiplexer_nav_when_zoomed = true,
+  -- Supply a Kitty remote control password if needed,
```

Comment:
By the way this seems unrelated to the rest of this PR, depending on how you feel about commit history and things you may want to have this be in a separate commit

Reply:
Yeah, the internals already supported this, it just wasn't documented, so this is fine.